include/kodlab_mjbots_sdk/attitude.h

```diff
@@ -0,0 +1,334 @@
+/**
+ * @file attitude.h
+ * @author Ethan Musser (emusser@seas.upenn.edu)
+ * @brief Defines an attitude object for storing inertial parameters.
+ * @date 7/7/22
+ * 
+ * @copyright 2022 The Trustees of the University of Pennsylvania. All rights reserved.
+ * 
+ */
+
+#pragma once
+
+#include <Eigen/Geometry>
+#include "kodlab_mjbots_sdk/pi3hat.h"
+#include "kodlab_mjbots_sdk/common_header.h"
+#include "kodlab_mjbots_sdk/rotations.h"
+#include <iostream>
+#include <iomanip>
+
+namespace kodlab
+{
+
+/**
+ * @brief Attitude object storing inertial data.
+ * @tparam Scalar[optional] numeric type
+ */
+template<typename Scalar = float>
+class Attitude
+{
+
+private:
+  /**
+   * @brief Inputted attitude quaternion before world offset rotation
+   */
+  Eigen::Quaternion<Scalar> att_quat_raw_ = {1, 0, 0, 0};
```

Comment:
IMO "att_..." is redundant, not a huge deal either way and perhaps slightly more correct when compared to the other things in this class, but I think it is ok for quat_,euler_,rot_mat. to be understood to be the attitude, where as

Reply:
Might be that the `Attitude` class is poorly named and should be something more akin to `IMUData`.  But, I think `att_...` is just explicitly identifying the variable data, and matches the convention for the rest of the class variables.

Reply:
Class name changed to `IMUData` and header name changed to `imu_data.h` in 2fe609067d9ac2f834dc243b9f5e117bbf844c36. Class members and accessors changed to remove `att_...` in 88098f1aa2c90661cd905c056e1c3bd4b1044d2b.  Marking this as resolved.

Reply:
Perhaps an attitude specific struct could exist in the future for added clarity, but for now this LGTM.